wordpress-mobile/WordPress-iOS
# Pages List: check for date value before inserting into array

Fixes #13737

The crash was caused by attempting to insert a `nil` date into an array, so I added a check to ensure the date has a value before attempting the insert. I couldn't figure out how a post would have a `nil` date, but alas it should no longer crash.

To test:
Since there isn't really a visible or functional change, just access all the tabs on `Site Pages` and verify the post dates appear.

PR submission checklist:
